features/promotion/managing_catalog_promotions/validating_catalog_promotion_creation.feature

```diff
@@ -153,7 +153,7 @@ Feature: Validating a catalog promotion creation
         And I add scope that applies on variants "PHP T-Shirt" variant and "Kotlin T-Shirt" variant
         And I add invalid fixed discount action configured for nonexistent channel
         And I try to add it
-        Then I should be notified that at least one of the provided channel codes does not exist
+        Then I should be notified that not all channels are filled
```

Comment:
Instead of changing the `Then` step, adjust this scenario to check the same result as before